omero_mapr/utils/version.py

```diff
@@ -30,12 +30,7 @@ def get_version(version=None):
     See https://www.python.org/dev/peps/pep-0440/
     """
 
-    version = get_full_version(version)
-    parts = 2 if version[2] == 0 else 3
-    res = '.'.join(str(x) for x in version[:parts])
-    if len(version) > 3:
-        res = "%s%s" % (res, version[3])
-    return str(res)
+    return get_full_version(version)
```

Comment:
The only other place where this API is being used is `custom_context_processor` but I don't know more about the context so I cannot comment on whether these changes would affect the usage

Reply:
```
def mapr_url_suffix(request):
    suffix = u"?_%s" % get_version()
    return {'mapr_url_suffix': suffix}
```

Reply:
I don't think it will be anything

Reply:
:+1: happy to follow your lead here.